frontend/src/views/Lobby.tsx

```diff
@@ -330,88 +396,116 @@ function LobbyPage() {
-
-      <MediumText>Difficulty Settings</MediumText>
-      {Object.keys(Difficulty).map((key) => {
-        const difficultyKey: Difficulty = Difficulty[key as keyof typeof Difficulty];
-        return (
-          <DifficultyButton
-            difficulty={difficultyKey}
-            onClick={() => updateDifficultySetting(key)}
-            active={difficulty === difficultyKey}
-            enabled={isHost(currentUser)}
-            title={!isHost(currentUser) ? 'Only the host can change these settings' : undefined}
+    <>
+      <CopyIndicatorContainer copied={copiedRoomLink}>
+        <CopyIndicator onClick={() => setCopiedRoomLink(false)}>
+          Link copied!&nbsp;&nbsp;✕
+        </CopyIndicator>
+      </CopyIndicatorContainer>
+      <HeaderContainer>
+        <MainHeaderText>
+          Join with the link
+          {' '}
+          <InlineBackgroundCopyText
+            onClick={() => {
+              copy(`https://codejoust.co/play?room=${currentRoomId}`);
+              setCopiedRoomLink(true);
+            }}
           >
-            {key}
-          </DifficultyButton>
-        );
-      })}
-
-      <MediumText>Duration</MediumText>
-      <Text>
-        {isHost(currentUser) ? 'Choose a game duration between 1-60 minutes:'
-          : 'The game will last for the following minutes:'}
-      </Text>
-      <NumberInput
-        min={1}
-        max={60}
-        value={duration}
-        disabled={!isHost(currentUser)}
-        onKeyPress={(e) => {
-          // Prevent user from using any of these non-numeric characters
-          if (e.key === 'e' || e.key === '.' || e.key === '-') {
-            e.preventDefault();
-          }
-        }}
-        onChange={(e) => {
-          const { value } = e.target;
-
-          // Set duration to undefined to allow users to clear field
-          if (!value) {
-            setDuration(undefined);
-          } else {
-            const newDuration = Number(value);
-            if (newDuration >= 0 && newDuration <= 60) {
-              setDuration(newDuration);
+            codejoust.co/play?room=
+            {currentRoomId}
+            <InlineCopyIcon>content_copy</InlineCopyIcon>
+          </InlineBackgroundCopyText>
+          {' '}
+          or at
+          {' '}
+          <i>codejoust.co/play</i>
+          {' '}
+          with Room ID:
+        </MainHeaderText>
+        <IdContainer id={currentRoomId} />
+        <PrimaryButtonZeroLeftMargin
+          onClick={handleStartGame}
+          disabled={loading || !isHost(currentUser)}
+          title={!isHost(currentUser) ? 'Only the host can start the game' : undefined}
+        >
+          Start Game
+        </PrimaryButtonZeroLeftMargin>
+      </HeaderContainer>
+
+      <FlexBareContainerLeft>
+        <PlayersContainer>
+          <SmallHeaderTextZeroTopMargin>
+            Players
+            {
+              (activeUsers && inactiveUsers)
+                ? ` (${activeUsers.length + inactiveUsers.length})`
```

Comment:
nit: I think the roomdto still passes the original list of users, so you might be able to just return users.length instead